migration/template/apply/20-bootstrap.sh

```diff
@@ -6,13 +6,26 @@ ROOT="$(readlink -f "$(dirname "${0}")/../../../")"
 source "${ROOT}/scripts/migration/lib.sh"
 
 run() {
+  local cluster="${2:-both}"
+
   case "${1:-}" in
   execute)
-    "${ROOT}/bin/ck8s" bootstrap sc
-    "${ROOT}/bin/ck8s" bootstrap wc
+    if [[ "${cluster:-}" =~ ^(sc|both)$ ]]; then
+      "${ROOT}/bin/ck8s" bootstrap sc
+    fi
+    if [[ "${cluster:-}" =~ ^(wc|both)$ ]]; then
+      "${ROOT}/bin/ck8s" bootstrap wc
+    fi
     ;;
   rollback)
     log_warn "rollback not implemented"
+
+    # if [[ "${cluster:-}" =~ ^(sc|both)$ ]]; then
+    #   log_info "rollback operation on service cluster"
+    # fi
+    # if [[ "${cluster:-}" =~ ^(wc|both)$ ]]; then
+    #   log_info "rollback operation on workload cluster"
+    # fi
```

Comment:
I think you can keep this in the template file but delete it in all the other files if it has not been implemented.
```suggestion
```